vagrant/provisioning/roles/arkcase-post-upgrade/tasks/main.yml

```diff
@@ -1,20 +1,32 @@
+- name: Get folders in a folder
+  find:
+    paths: "{{ root_folder }}/data/arkcase-home/.arkcase-backup*"
+  register: found_folders
+
+- name: Get latest folder
+  set_fact:
```

Comment:
what will happen if there is no folder "{{ root_folder }}/data/arkcase-home/.arkcase-backup*", or if there are no backups in the folder, or if there is a backup folder but it doesn't have these files?